newswires/app/db/FingerpostWireEntry.scala

```diff
@@ -161,8 +161,8 @@ object FingerpostWireEntry
   }
 
   case class QueryResponse(
-      results: List[FingerpostWireEntry],
```

Comment:
I've opted to delete the code downstream, in rendering logic etc., but comment it out at the top of the stream. I'm not sure why but this feels like a reasonable balance..? We should delete it later on though if we aren't going to reinstate for a while.